### PR TITLE
PACT 931 and PACT-932: refactor of services and integration tests

### PR DESCRIPTION
**Pull request by rwalpole**

### CODE REVIEW

File: src/it/scala/MessageRecoveryISpec.scala [hunk 1]
```diff
@@ -1,6 +1,8 @@
+import cats.effect
```
Comment (userjd) [inline src/it/scala/MessageRecoveryISpec.scala:1]: unused import

File: src/it/scala/MessageRecoveryISpec.scala [hunk 2]
```diff
@@ -9,10 +11,10 @@ import jms4s.sqs.simpleQueueService.{ Config, Credentials, DirectAddress, HTTP }
 import org.apache.commons.lang3.SerializationUtils
 import org.scalatest.TryValues.convertTryToSuccessOrFailure
 import org.scalatest.concurrent.{ Eventually, IntegrationPatience }
-import org.scalatest.freespec.AsyncFreeSpec
+import org.scalatest.freespec.{ AsyncFreeSpec, FixtureAsyncFreeSpec }
```
Comment (userjd) [inline src/it/scala/MessageRecoveryISpec.scala:14]: unused import

File: src/it/scala/MessageRecoveryISpec.scala (hunk 1 above)
Comment (rwalpole) [reply src/it/scala/MessageRecoveryISpec.scala]: Removed.

File: src/it/scala/MessageRecoveryISpec.scala (hunk 2 above)
Comment (rwalpole) [reply src/it/scala/MessageRecoveryISpec.scala]: Removed.

File: src/it/scala/MessageRecoveryISpec.scala [hunk 3]
```diff
@@ -22,7 +24,7 @@ import uk.gov.nationalarchives.omega.api.services.ApiService
 
 import java.nio.file._
 import java.util.UUID
-import scala.concurrent.Await
+import scala.concurrent.Future
```
[the remark this answers is not in the document]
Comment (rwalpole) [reply src/it/scala/MessageRecoveryISpec.scala]: Removed.

**Review by userjd — APPROVED** [review]

**Pull request merged**